specification/hdinsight/resource-manager/Microsoft.HDInsight/2015-03-01-preview/cluster.json

```diff
@@ -96,62 +93,12 @@
                 },
                 "x-ms-long-running-operation": true
             },
-            "patch": {
-                "tags": [
-                    "Clusters"
-                ],
-                "operationId": "Clusters_Update",

```

Comment:
Why this operation is removed?

(1). Is it the case that RP never supported this operation hence having this operation in the swagger is simply a bug in the spec and you are just fixing it? OR (2). RP used to support this operation but not supported anymore?

If (2) is the case then it is a breaking change, which requires api-version upgrade. Please confirm.

Reply:
Hmm, good catch. This must have been an error in the commit, since we still support patching.